misk-redis/src/main/kotlin/misk/redis/FakeRedis.kt

```diff
@@ -200,19 +200,21 @@ class FakeRedis : Redis {
     }
   }
 
-  override fun hset(key: String, field: String, value: ByteString) {
+  override fun hset(key: String, field: String, value: ByteString): Long {
     if (!hKeyValueStore.containsKey(key)) {
       hKeyValueStore[key] = Value(
         data = ConcurrentHashMap(),
         expiryInstant = Instant.MAX
       )
     }
+    val newFieldCount = if (hKeyValueStore[key]!!.data[field] != null) { 0L } else 1L
```

Comment:
nit: `if (hKeyValueStore[key]!!.data.containsKey(field)) 0L else 1L` (mostly just remove the brackets around the `0L`